selfdrive/ui/translations/main_nl.ts

```diff
@@ -0,0 +1,1307 @@
+        <source>Enable Lane Departure Warnings</source>
+        <translation>Waarschuwingen bij Verlaten Rijstrook Inschakelen</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="41"/>
+        <source>Receive alerts to steer back into the lane when your vehicle drifts over a detected lane line without a turn signal activated while driving over 31 mph (50 km/h).</source>
+        <translation>Ontvang waarschuwingen om terug naar de rijstrook te sturen wanneer uw voertuig over een gedetecteerde rijstrookstreep drijft zonder dat de richtingaanwijzer wordt geactiveerd terwijl u harder rijdt dan 50 km/u (31 mph).</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="46"/>
+        <source>Use Metric System</source>
+        <translation>Gebruik Metrisch Systeem</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="47"/>
+        <source>Display speed in km/h instead of mph.</source>
+        <translation>Geef snelheid weer in km/u in plaats van mph.</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="52"/>
+        <source>Record and Upload Driver Camera</source>
+        <translation>Opnemen en Uploaden van de Bestuurders Camera</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="53"/>
+        <source>Upload data from the driver facing camera and help improve the driver monitoring algorithm.</source>
+        <translation>Upload gegevens van de bestuurders camera en help het algoritme voor het monitoren van de bestuurder te verbeteren.</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="58"/>
+        <source>Disengage On Accelerator Pedal</source>
+        <translation>Deactiveren Met Gaspedaal</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="59"/>
+        <source>When enabled, pressing the accelerator pedal will disengage openpilot.</source>
+        <translation>Indien ingeschakeld, zal het indrukken van het gaspedaal openpilot deactiveren.</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="65"/>
+        <source>Show ETA in 24h format</source>
+        <translation>Toon verwachte aankomsttijd in 24-uurs formaat</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="66"/>
+        <source>Use 24h format instead of am/pm</source>
+        <translation>Gebruik 24-uurs formaat in plaats van AM en PM</translation>
+    </message>
+    <message>
+        <location filename="../qt/offroad/settings.cc" line="71"/>
+        <source>Show Map on Left Side of UI</source>
+        <translation>Toon kaart aan de linkerkant van de gebruikersinterface</translation>
```

Comment:
Is there a shorter word for `gebruikersinterface`? This makes the toggles pane horizontally scrollable

Reply:
Can you verify the 2 changes?